src/Payments.Mvc/Views/Teams/ConfirmAccount.cshtml

```diff
@@ -0,0 +1,129 @@
+                <input asp-for="Description" class="form-control" />
+                <span asp-validation-for="Description" class="text-danger"></span>
+            </div>
+            <dl class="dl-horizontal">
+                <dt>
+                    @Html.DisplayNameFor(model => model.Chart)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Chart)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.Account)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Account)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.Object)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Object)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.SubAccount)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.SubAccount)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.SubObject)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.SubObject)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.Project)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.Project)
+                </dd>
+                <dt>
+                    @Html.DisplayNameFor(model => model.IsDefault)
+                </dt>
+                <dd>
+                    @Html.DisplayFor(model => model.IsDefault)
+                </dd>
+            </dl>
+            <input type="hidden" asp-for="Chart" />
+            <input type="hidden" asp-for="Account" />
+            <input type="hidden" asp-for="Object" />
+            <input type="hidden" asp-for="SubAccount" />
+            <input type="hidden" asp-for="SubObject" />
+            <input type="hidden" asp-for="Project" />
+            <input type="hidden" asp-for="IsDefault" />
+
+
+            <input type="hidden" id="confirm" name="confirm" value="true" />
+            <div class="form-group">
+                <input type="submit" value="Confirm" class="btn btn-default" />
+                <a class="btn btn-secondary" id="edit">Edit</a>
+            </div>
+        </form>
+    </div>
+</div>
+
+<div>
+    <a asp-action="Index">Back to List</a>
+</div>
+
+@section Scripts {
+    @{await Html.RenderPartialAsync("_ValidationScriptsPartial");}
+    <script type="text/javascript">
+
+        $('#edit').on('click',
+            function() {
+                $("#confirm").val("false");
+                $("#confirm-create").submit();
+            });
+
+
+    </script>
+
+    @*<script type="text/javascript">
+            $("#Account").on("blur",

```

Comment:
@jSylvestre actually this whole are should be gone right?  account is a hidden field here and can never be blurred so this can't be used